src/main/java/com/google/sps/servlets/MemesServlet.java

```diff
@@ -87,5 +92,40 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       String imgTag = String.format("<img src=\"%s\" width = \"300\"/>", blob.getMediaLink());
       response.getWriter().println(imgTag);
     }
+  }
+
+/** This funtions  randomly  get a string  captions from an array of string. When the function is called
+ * it randomly return memes caption from an array
+ * input: null
+ * return: strings..
+ * 
+ */
+  
+
+  public static String getCpation(){
```

Comment:
Curious, why did you choose to make this "public static"? An alternative is making it a "private" helper function.